src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -8,6 +9,8 @@
 from sentry.utils import json, kafka_config, metrics
 from sentry.utils.pubsub import KafkaPublisher
 
+logger = logging.getLogger()
```

Comment:
it's worth adding a name for the logger so it's easier to find in GCP https://github.com/getsentry/sentry/blob/6d9101b39c04d75f415aa523e9c82c5444de7ca5/src/sentry/replays/usecases/ingest/__init__.py#L23